package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.6.0",
   "description": "A web framework for building virtual reality experiences.",
   "homepage": "https://aframe.io/",
-  "main": "dist/aframe-master.js",
+  "main": "./dist/aframe-master.js",
+  "module": "./dist/aframe-master.module.min.js",
```

Comment:
The `"module"` field never really became standardized, instead we should use `"exports"` (https://webpack.js.org/guides/package-exports/). Luckily most tooling can handle things gracefully, so we can keep `module` in there as well for older tools that might not support `exports` yet.

Reply:
good point, I added exports